src/WebJobs.Script.WebHost/Diagnostics/DiagnosticListenerService.cs

```diff
@@ -63,7 +65,7 @@ private void SubscribeListeners()
                             _listenerSubscriptions.Add(listener.Name, listenerSubscription);
                         }
 
-                        _logger.LogInformation("Subscribed to host diagnostic source '{sourceName}'", listener.Name);
+                        _logger.LogInformation("Subscribed to diagnostic source '{sourceName}'", listener.Name);
```

Comment:
Can you double check that changing this log won't impact any detectors?

Reply:
Thanks for the reminder, Lilian! I just took a look through the detectors repository, and from what I see, there does not appear to be a detector that searches explicitly for the string `"Subscribed to host diagnostic..."`, so I think this change shouldn't break anything on that front.

Reply:
Thanks for checking!